### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by raikusy**
# Docker setup is complete.
## To run project - 
`docker-compose up`
## To stop project - 
`docker-compose down`

## Reference

- [dockerizing-django-with-postgres-gunicorn-and-nginx](https://testdriven.io/blog/dockerizing-django-with-postgres-gunicorn-and-nginx/)
- [building-django-docker-image-with-alpine-32de65d2706](https://medium.com/c0d1um/building-django-docker-image-with-alpine-32de65d2706)

## To run command inside project -
`docker-compose exec web python manage.py migrate --noinput`
Simply put any python cmd after `docker-compose exec web`

Cheers! 🥇

### CODE REVIEW

File: course/migrations/0004_auto_20191214_2233.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.7 on 2019-12-14 22:33
+
```
Comment (tanvirstreame) [inline course/migrations/0004_auto_20191214_2233.py]: Does this migration is part of docker setup?

File: course/models.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ def __str__(self):
 
 
 class CourseAttendance(models.Model):
-    date = models.DateTimeField(default=datetime.now(), blank=True)
+    date = models.DateTimeField(auto_now_add=True, blank=True)
```
Comment (tanvirstreame) [inline course/models.py]: You have changes in models , so migration is needed

File: course/migrations/0004_auto_20191214_2233.py (hunk 1 above)
Comment (raikusy) [reply course/migrations/0004_auto_20191214_2233.py]: This file is auto generated. Docker will run migration command each time when building image. but not when you run the image.

Comment (tanvirstreame) [reply course/migrations/0004_auto_20191214_2233.py]: I see models has been changes..... Look add the date field changes in the pull request. No problem of date field changes....
Good work by the way :)

Comment (raikusy) [reply course/migrations/0004_auto_20191214_2233.py]: Well I don't know django much. I just added docker :)
Thank you!
